app/Http/Controllers/AppRegisterController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\AppRegister;
+use App\Mail\InAppRegisterEmailVerify;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Mail;
+
+class AppRegisterController extends Controller
+{
+    public function index(Request $request)
+    {
+        abort_unless(config('auth.iar') == true, 404);
+        // $open = (bool) config_cache('pixelfed.open_registration');
+        // if(!$open || $request->user()) {
```

Comment:
nit: remove dead code if its not going to be used

Reply:
This isn't dead, just temporary while I test on the closed staging server.